Neos.Eel/Classes/CompilingEvaluator.php

```diff
@@ -103,7 +103,9 @@ protected function generateEvaluatorCode($expression)
 
         if ($result === false) {
             throw new ParserException(sprintf('Expression "%s" could not be parsed.', $expression), 1344513194);
+            /** @phpstan-ignore-next-line */
```

Comment:
```suggestion
            /** @phpstan-ignore-next-line $parser->pos appears to be undeclared, because instantiation of $parser is hidden as well */
```

I believe we should comment each occurrence of `/** @phpstan-ignore-next-line */` to give future people (including our future selves) a fair chance to understand and resolve the issue eventually.

L100 would need treatment as well, though:
```php
/** @phpstan-ignore-next-line `CompilingEelParser` has been explicitly excluded from analysis, because it contains generated code that is impossible to fix */
```

(Don't know if my assumption in that comment is entirely correct though)

Reply:
Yes good idea, well anything regarding the parsers is a lost cause :D